src/app/seamly2d/mainwindow.cpp

```diff
@@ -7161,20 +7164,20 @@ bool MainWindow::IgnoreLocking(int error, const QString &path)
             case QLockFile::LockFailedError:
                 answer = QMessageBox::warning(this, tr("Locking file"),
                                                tr("This file already opened in another window. Ignore if you want "
-                                                  "to continue (not recommended, can cause a data corruption)."),
+                                                  "to continue (!recommended, can cause a data corruption)."),
                                                QMessageBox::Abort|QMessageBox::Ignore, QMessageBox::Abort);
                 break;
             case QLockFile::PermissionError:
                 answer = QMessageBox::question(this, tr("Locking file"),
                                                tr("The lock file could not be created, for lack of permissions. "
-                                                  "Ignore if you want to continue (not recommended, can cause "
+                                                  "Ignore if you want to continue (!recommended, can cause "
                                                   "a data corruption)."),
                                                QMessageBox::Abort|QMessageBox::Ignore, QMessageBox::Abort);
                 break;
             case QLockFile::UnknownError:
                 answer = QMessageBox::question(this, tr("Locking file"),
                                                tr("Unknown error happened, for instance a full partition prevented "
-                                                  "writing out the lock file. Ignore if you want to continue (not "
+                                                  "writing out the lock file. Ignore if you want to continue (!"
```

Comment:
I dont think you wanted to change the error message here

Reply:
ditto.